src/stdlib_math.fypp

```diff
@@ -42,6 +42,18 @@ module stdlib_math
     #:endfor
     end interface clip
 
+    interface swap
```

Comment:
```suggestion
    !> Returns the greatest common divisor of two integers
    !> ([Specification](../page/specs/stdlib_math.html#swap_subroutine))
    !>
    !> Version: experimental
    interface swap
```

Reply:
Maybe this as alternative: https://github.com/fortran-lang/stdlib/pull/869/commits/6a30b2ff974aaa9a0a27aa730e0ece321c8b13ac ?